boardgamegeek/legacy_api.py

```diff
@@ -0,0 +1,55 @@
+import logging
+
+# import xml.etree.ElementTree as ET
+
+from .api import BGGCommon
+from .api import CacheBackendMemory
+from .api import DEFAULT_REQUESTS_PER_MINUTE
+from .api import BGGValueError
+from .api import request_and_parse_xml
+from .loaders import create_geeklist_from_xml, add_geeklist_items_from_xml
+
+log = logging.getLogger("boardgamegeek.legacy_api")
+
+API_ENDPOINT='http://www.boardgamegeek.com/xmlapi'
+
+class BGGClientLegacy(BGGCommon):
+    def __init__(self, cache=CacheBackendMemory(ttl=3600), timeout=15, retries=3, retry_delay=5, disable_ssl=False, requests_per_minute=DEFAULT_REQUESTS_PER_MINUTE):
+
+        super(BGGClientLegacy, self).__init__(api_endpoint=API_ENDPOINT,
+                                              cache=cache,
+                                              timeout=timeout,
+                                              retries=retries,
+                                              retry_delay=retry_delay,
+                                              requests_per_minute=requests_per_minute)
+        self._search_api_url = None
+        self._thing_api_url = None
+        self._guild_api_url = None
+        self._user_api_url = None
+        self._plays_api_url = None
+        self._hot_api_url = None
+        self._collection_api_url = None
+        self._geeklist_api_url = API_ENDPOINT + "/geeklist"
+
+
+    def geeklist(self, listid, comments=False):
+        # Parameter validation
+        if not listid:
+            raise BGGValueError("List Id must be specified")
+        log.debug("retrieving list {}".format(listid))
+
+        params = {}
+        if comments: params['comments']= 1
+        url = "%s/%s" % (self._geeklist_api_url, listid)
```

Comment:
code style consistency: use .format()

Reply:
Fixed in d3c51c51f9f90d754a42e1653bd928dd4b14e00e